torch/_refs/__init__.py

```diff
@@ -1654,6 +1655,27 @@ def item(a: TensorLikeType) -> NumberType:
     return number_type(prims.item(a))
 
 
+@register_decomposition(torch.ops.aten.to)
+def to(
+    a: TensorLikeType,
+    dtype: torch.dtype,
+    non_blocking: bool = False,
+    copy: bool = False,
+    memory_format: torch.memory_format = torch.preserve_format,
+) -> TensorLikeType:
+    if (
+        (copy is True or dtype != a.dtype)
+        and memory_format == torch.preserve_format
+        and non_blocking is False
+    ):
+        return torch.ops.nvprims.convert_element_type(a, dtype)
```

Comment:
_refs module should use plain prims
```suggestion
        return torch.ops.prims.convert_element_type(a, dtype)
```